packaging/amazon-linux-ami/ipCleanup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+sysctl net.ipv4.conf.default.route_localnet
+sysctl -w net.ipv4.conf.all.route_localnet=0
+iptables -t nat -D PREROUTING -p tcp -d 169.254.170.2 --dport 80 -j DNAT --to-destination 127.0.0.1:51679
+
+if [ -n "${ECS_SKIP_LOCALHOST_TRAFFIC_FILTER}" ]
```

Comment:
Please correct me if I am wrong. Do we only need to handle the case when `ECS_SKIP_LOCALHOST_TRAFFIC_FILTER` is specified to `true`? Because `ECS_SKIP_LOCALHOST_TRAFFIC_FILTER` is set to `false` as default in [ecs-init](https://github.com/aws/amazon-ecs-init). 

Same question for `ECS_ALLOW_OFFHOST_INTROSPECTION_ACCESS`. Thanks.

Reply:
You're right, but the script will get stuck if either of the environment variables are unset. So I am first testing to see if it is set or not before comparing it to see whether it is true. If there is a better way to test for this, I am all ears. Hopefully that clears it up a bit...